src/simulator/start_simulator.py

```diff
@@ -20,16 +20,14 @@
 SERVICE_NAME = "full_node"
 
 
-def service_kwargs_for_full_node_simulator(
-    root_path: Path,
-    config: Dict,
-    bt: BlockTools,
-) -> Dict:
+def service_kwargs_for_full_node_simulator(root_path: Path, config: Dict, bt: BlockTools, constants_dic=None) -> Dict:
```

Comment:
This doesn't seem great. Can we instead replace the field in block tools constants?

Reply:
fixed, moved to setup nodes / blocktools